src/content/en/fundamentals/instant-and-offline/get-started/graphicalcontent-4.md

```diff
@@ -0,0 +1,284 @@
+of Exif removal at MakeUseOf.
+
+[http://www.makeuseof.com/tag/3-ways-to-remove-exif-metadata-from-photos-and-why-you-might-want-to/](http://www.makeuseof.com/tag/3-ways-to-remove-exif-metadata-from-photos-and-why-you-might-want-to/)
+
+##Resize Images
+
+###Size images based on their intended use
+
+Large images take longer to download than smaller ones. All your images should be appropriately 
+sized for their intended use and should not rely on the browser to resize them for rendering.
+
+Often, large images are retrieved from the server and then resized in the browser with CSS. 
+This might be for a perfectly logical reason -- such as using just one image for a thumbnail 
+that expands on hover -- or it might just be careless coding. Whatever the reason, it wastes 
+bandwidth.
+
+For example, you might have a 1200 x 600 pixel image that you present at 60 x 30 (as a 5% 
+thumbnail) and roll it up to full size on hover using a CSS transition. It works fine and looks 
+great, but if the user never actually hovers over the thumbnail, then 95% of the time that 
+image took to download was wasted. You may be better off taking the time to make separate, 
+properly sized thumbnails, displaying the full sized images only if they're actually requested.
+
+###Crop images to show only what's important
+
+One enormously effective technique to reduce file size for images of all kinds is simple 
+cropping: omitting parts of the image that aren't important to information delivery.
+
+Extra background and whitespace, unnecessary borders, and unintended objects clutter an image 
+not just visually but physically, affecting the image size and its download efficiency.
+
+The photo of the Australia traveler shown above is, frankly, too large, both horizontally and 
+vertically. It could be cropped to more effectively focus on the person, the sign on the fence, 
+and the main part of the house.
+
+![bonniedoon.jpg, cropped, 115k](images/image_414.jpg)
+*bonniedoon.jpg, cropped, 115k*
+
+Here, we started with the already-resized version starting at 220k, then cropped it, 
+saving it to the version seen above at 115k. That's a 52% reduction, and the photo now 
+focuses on the important parts (yes, the doghouse is important) at half the file size. 
+All the required information is there, it's just more efficiently presented.
+
+###Reduce image quality
+
+Of course, we want our images to look as good as possible, within reason, so that means saving 
+JPGs at 100% quality, right? Not necessarily. In most (which is to say nearly all) cases, 
+you can reduce the JPG quality, and thus the file size, without suffering any visible quality 
+difference.
+
+Once you've determined the size that an image should really be, how do you get it to that size 
+but still retain the quality and characteristics it needs? The first thing you need is a good 
+image manipulation tool. 
+[ResizeIt](https://itunes.apple.com/us/app/resizeit/id416280139?mt=12) 
```

Comment:
Move resize it reference down to additional tools, and make the windows tool the one that you are using.